service/src/main/kotlin/xyz/lebkuchenfm/api/auth/AuthEndpoint.kt

```diff
@@ -0,0 +1,45 @@
+package xyz.lebkuchenfm.api.auth
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.auth.authenticate
+import io.ktor.server.auth.principal
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import io.ktor.server.sessions.clear
+import io.ktor.server.sessions.get
+import io.ktor.server.sessions.sessions
+import io.ktor.server.sessions.set
+import kotlinx.serialization.Serializable
+import xyz.lebkuchenfm.domain.auth.UserSession
+
+fun Route.authRouting() {
+    route("/auth") {
+        get {
+            val session = call.sessions.get<UserSession>()!!
```

Comment:
force?

Reply:
Yes. Technically it should never be null. So at least while we're in development mode I would like it to throw Null Pointer Exception so we can debug the case when it is.